.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    env:
+      CHROME_EXTENSION_ID: dclfaikcemljbaoagjnedmlppnbiljen
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 16
+      - name: Install Dependencies
+        shell: bash
+        run: npm install
+      - name: Set environment variables from the manifest
+        run: |
+          echo "FIREFOX_EXTENSION_ID=$(jq -s -r ".[0].browser_specific_settings.gecko.id" < src/manifest_firefox.json)" >> "$GITHUB_ENV"
+          echo "CURRENT_VERSION=$(jq -s -r '.version' < src/manifest.json)" >> "$GITHUB_ENV"
+      - name: Check if Firefox version can be released
+        id: firefox-check
```

Comment:
I'm pre-empting some issues with calling `steps.firefox-check.outcome` (might need to be `steps["firefox-check"].outcome` etc) - I'm not sure how well GitHub Actions copes with kebab-cased IDs. The [examples](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#steps-context) in the docs use snake_case.